composer.json

```diff
@@ -9,10 +9,10 @@
         }
     ],
     "require": {
-        "php": ">=8.2"
+        "php": ">=8.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~11.0",
+        "phpunit/phpunit": "^10",
```

Comment:
```suggestion
        "phpunit/phpunit": "^10 || ^11",
```

I think it would make sense to support both versions, so that this library does not support an upgrade to PHPUnit 11. Wdyt?